src/screens/NewAnalytics/PaymentAnalytics/PaymentsLifeCycle/PaymentsLifeCycleUtils.res

```diff
@@ -34,7 +34,7 @@ let paymentsLifeCycleMapper = (
   let customerAwaited = data.customerAwaited // DropOff2
   let attemptedPayments = pending + customerAwaited + success + failure
   let pmAwaited = data.pmAwaited // Dropoff1
-  let totalPayment = pmAwaited + attemptedPayments + cancelled
+  let _totalPayment = pmAwaited + attemptedPayments + cancelled
```

Comment:
If we dont need this then can we remove this ?

Reply:
we will remove this